apps/web/src/design-system/table/Table.tsx

```diff
@@ -116,7 +116,7 @@ export function Table<T extends object>({
   const noData = rows.length === 0;
 
   return (
-    <div style={{ position: 'relative', minHeight: 500, display: 'flex', flexDirection: 'column' }}>
+    <div style={{ position: 'relative', minHeight: 500, display: 'flex', flexDirection: 'column', height: '85%' }}>
```

Comment:
added height: 85% for now to show the pagination at the bottom of screen.
once we implement infinite scroll we might need to adjust this value